examples/BLDC_HALL/README.md

```diff
@@ -0,0 +1,9 @@
+# Setup for Brushlessmotor with Hallsensor
+
+Status: not tested, not working
```

Comment:
Jeah, I think you also have to change a few resistors. I would also mention this in the code, because I'm pretty sure a few people will still don't do this, even though it is mentioned in the "Getting Started".